governance/polls/MIP10c14-SP7: Appoint DeFi Saver as Oracle Light Feed.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Appoint DeFi Saver as Oracle Light Feed (MIP10c14-SP7) - April 5, 2021
+summary: Signal your support or opposition for appointing DeFi Saver as Oracle Light Feed
+discussion_link: https://forum.makerdao.com/t/mip10c14-sp7-proposal-appoint-defi-saver-as-a-light-feed/7207
+vote_type: Plurality Voting
+categories:
+   - Oracles
+options:
+   0: Abstain
+   1: Yes
+   2: No
+---
+# Appoint DeFi Saver as Oracle Light Feed (MIP10c14-SP7) - April 5, 2021
+
+The Governance Facilitators have placed a Governance Poll into the voting system on behalf of DeFi Saver. This Governance Poll will be active for three days beginning on Monday, April 5 at 16:00 UTC.
+
+Note: This subproposal has been submitted under the informal weekly cycle rather than the MIP3 Governance Cycle due to inconsistencies between MIP10 and MIP3 and to avoid delaying progress in the oracles domain until MIP10 can be amended.
+
+**This is a binary vote. In a binary vote, you may vote for a single option. You should vote for the option which you prefer. Alternatively, if you would accept either of the two options and have no preference between them, you should abstain.**
+
+## Review
+
+The community can vote in this poll to express support or opposition to appointing DeFi Saver as Oracle Light Feed.
```

Comment:
Thinking there should probably be some documentation here? MIP10 has this which is better than nothing:

Description Feeds are bots run by individuals and organizations that submit data to the Oracles. There are two types of Feeds; Dark Feeds run by anonymous individuals, and Light Feeds run by public organizations. This hybrid model preserves the hardness properties of Dark Feeds but benefits from the reputation of Light Feeds who are stakeholders in the ecosystem and are effectively staking their reputation. Feeds are paid a monthly Feed Stipend for the service they provide that is determined by governance.

Reply:
Maybe link to that section of MIP10?

Reply:
Good thinking, I'm linking to that section.